smibhid/lib/slack_api.py

```diff
@@ -35,6 +35,11 @@ async def async_get_space_state(self) -> bool | None:
             self.log.error(f"Unable to load space state from response data: {e}")
             raise
         return state
+    
+    async def async_upload_ui_log(self, log: list) -> None:
+        """Upload the UI log to the server."""
+        json_log = dumps({"log" : log})
+        await self._async_slack_api_request("POST", "ui_log", json_log)
```

Comment:
The method name? It could be, but as it's in the SMIBHID code base for sending SMIBHID data to SMIB that seems unnecessary to me? Have you got an example of how it can be confusing?

Reply:
I mean the api endpoint

Reply:
I mean the api endpoint

Reply:
Commit added for this.